### ISSUE TITLE
crash using command item replace

### ISSUE THREAD

**Issue by ykvox** [A5b84/armor-chroma-fabric#33]
hello !
using /item replace on an armor slot crashes the game

here are some logs
```
[15:43:33] [Render thread/ERROR]: Unreported exception thrown!
java.lang.IllegalArgumentException: Modifier is already applied on this attribute!
	at knot/net.minecraft.class_1324.method_6197(class_1324.java:77) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1324.method_26835(class_1324.java:93) ~[client-intermediary.jar:?]
	at knot/nukeduck.armorchroma.GuiArmor.lambda$getArmorPoints$0(GuiArmor.java:199) ~[armorchroma-1.2.8.jar:?]
	at knot/net.minecraft.class_9285.method_57482(class_9285.java:77) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1799.method_57354(class_1799.java:951) ~[client-intermediary.jar:?]
	at knot/nukeduck.armorchroma.GuiArmor.getArmorPoints(GuiArmor.java:197) ~[armorchroma-1.2.8.jar:?]
	at knot/nukeduck.armorchroma.GuiArmor.draw(GuiArmor.java:70) ~[armorchroma-1.2.8.jar:?]
	at knot/net.minecraft.class_329.handler$zdo000$armorchroma$onBeforeRenderArmor(class_329.java:6537) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_329.method_58478(class_329.java) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_329.method_1760(class_329.java:865) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_329.method_55805(class_329.java:541) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_9080.method_55813(class_9080.java:36) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_9080.method_55812(class_9080.java:22) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_9080.method_55813(class_9080.java:36) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_9080.method_55809(class_9080.java:29) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_329.method_1753(class_329.java:218) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_757.method_3192(class_757.java:896) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1523(class_310.java:1285) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1514(class_310.java:882) [client-intermediary.jar:?]
	at knot/net.minecraft.client.main.Main.main(Main.java:256) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.16.7.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
[15:43:33] [Render thread/INFO]: [voicechat] Clearing audio channels
[15:43:33] [Render thread/INFO]: [voicechat] Stopping microphone thread
[15:43:33] [Render thread/INFO]: [voicechat] Disconnecting voicechat
[15:43:33] [Render thread/INFO]: Unloading client level [Wrapped{ClientLevel@overworld}]-[overworld].
[15:43:33] [Render thread/INFO]: Unloading level com.seibel.distanthorizons.core.level.DhClientLevel@64c5979a
[15:43:33] [Render thread/INFO]: Shutting down ClientRenderState
[15:43:33] [Render thread/INFO]: Shutting down LodQuadTree...
[15:43:33] [Render thread/INFO]: Finished shutting down LodQuadTree
[15:43:34] [Render thread/INFO]: Closing [RemoteFullDataSourceProvider] for level: [com.seibel.distanthorizons.core.level.DhClientLevel@64c5979a].
[15:43:34] [Render thread/INFO]: Closing [FullDataSourceProviderV1] for level: [com.seibel.distanthorizons.core.level.DhClientLevel@64c5979a].
[15:43:34] [Render thread/INFO]: Closing database connection: [jdbc:sqlite:C:\Users\User\Desktop\PrismPortable\Data\instances\Ascension 1.21\.minecraft\Distant_Horizons_server_data\+\overworld\DistantHorizons.sqlite]
[15:43:34] [DH-Parent Update Queue [overworld] Thread[0]/INFO]: Update thread [DH-Parent Update Queue [overworld] Thread[0]] terminated.
[15:43:34] [Render thread/INFO]: Closed [DhClientLevel] for [Wrapped{ClientLevel@overworld}]
[15:43:34] [Render thread/INFO]: Client on ClientOnly mode disconnecting.
[15:43:34] [Render thread/INFO]: Closed DhWorld of type Client_Only
[15:43:34] [Render thread/INFO]: Closing all [0] database connections...
[15:43:34] [Render thread/INFO]: Stopping worker threads
[15:43:34] [Render thread/INFO]: [ETF]: emissive suffixes loaded: {_e}.
---- Minecraft Crash Report ----
// Don't do that.

Time: 2024-12-10 15:43:34
Description: Unexpected error

java.lang.IllegalArgumentException: Modifier is already applied on this attribute!
	at knot//net.minecraft.class_1324.method_6197(class_1324.java:77)
	at knot//net.minecraft.class_1324.method_26835(class_1324.java:93)
	at knot//nukeduck.armorchroma.GuiArmor.lambda$getArmorPoints$0(GuiArmor.java:199)
	at knot//net.minecraft.class_9285.method_57482(class_9285.java:77)
	at knot//net.minecraft.class_1799.method_57354(class_1799.java:951)
	at knot//nukeduck.armorchroma.GuiArmor.getArmorPoints(GuiArmor.java:197)
	at knot//nukeduck.armorchroma.GuiArmor.draw(GuiArmor.java:70)
	at knot//net.minecraft.class_329.handler$zdo000$armorchroma$onBeforeRenderArmor(class_329.java:6537)
	at knot//net.minecraft.class_329.method_58478(class_329.java)
	at knot//net.minecraft.class_329.method_1760(class_329.java:865)
	at knot//net.minecraft.class_329.method_55805(class_329.java:541)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55812(class_9080.java:22)
	at knot//net.minecraft.class_9080.method_55813(class_9080.java:36)
	at knot//net.minecraft.class_9080.method_55809(class_9080.java:29)
	at knot//net.minecraft.class_329.method_1753(class_329.java:218)
	at knot//net.minecraft.class_757.method_3192(class_757.java:896)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1285)
	at knot//net.minecraft.class_310.method_1514(class_310.java:882)
	at knot//net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
```

**Comment by A5b84** [comment]
Will be fixed in the next version (was fixed in commit 94a80029)

**Comment by ykvox** [comment]
fantastic ! thank you so much
